### DISCUSSION TITLE
NSAdvertisingAttributionReportEndpoint - how to have two urls?

### DISCUSSION THREAD

**Discussion by shaymolho** [airsdk/Adobe-Runtime-Support#2756] [Q&A]
I use both appsflyer and iron source
appsflyer requires this to be:
```
<key>NSAdvertisingAttributionReportEndpoint</key>
<string>https://appsflyer-skadnetwork.com/</string>
```

while iron source requires this to be
```
<key>NSAdvertisingAttributionReportEndpoint</key>
<string>https://postbacks-is.com</string>
```

Is there a way to include both of them?
If no, what is the suggested solution?

**Comment by marchbold** [comment]
This is a single value you cannot support multiple endpoints from a device. You may be able to setup a server to act as an intermediary for them though.
